### ISSUE TITLE
Test for WCAG criteria

### ISSUE THREAD

**Issue by klues** [asterics/AsTeRICS-Grid#31]
Although I already tried to respect the most important aspects of WCAG, there should be a separate in-depth test of the WCAG criteria: https://www.w3.org/TR/WCAG21/

Test should be done on the current public version: https://grid.asterics.eu/

**Comment by willwade** [comment]
Just an idea..

- It definitely needs manual looking at that this. I think there may be some web accessibility groups we can lean on for this..
- But automated wise we would be wise integrating axe-core into the Vue.js component tests to catch accessibility issues before they are even committed.
- Also set up Pa11y to run a full automated scan on the publicly deployed version of grid.asterics.eu with every new release. 

(NB: there is a lighthouse too which uses axe-core but I think this would be a better approach)

**Comment by ms-mialingvo** [comment]
I was experimenting with Stark some days ago for reasons unrelated to AG. If it's of any use, here are reports of AG. https://we.tl/t-7mehCdZZoI  Overall it looks good, though I don't know how reliable the report is.
